packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/network/ResponseUtil.kt

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.modules.network
+
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.WritableMap
+import java.net.SocketTimeoutException
+
+/** Util methods to send network responses to JS. */
+public object ResponseUtil {
+  @JvmStatic
+  public fun onDataSend(
+    reactContext: ReactApplicationContext?, requestId: Int, progress: Long, total: Long
+  ) {
+    val args = Arguments.createArray()
```

Comment:
Commenting here but valid for the whole class:

Can you change to this:

```suggestion
    val args = Arguments.createArray().apply {
        pushInt(requestId)
        pushInt(progress.toInt())
        pushInt(total.toInt())
    }
```

Reply:
Absolutely! This has been addressed in add11d10463f71ffde711b0013e562fdcb8bc657. Thanks for pointing out